src/commands/login.ts

```diff
@@ -32,6 +38,8 @@ export const login = async (
   args: { org: string },
   options?: { skipAuthenticate?: boolean }
 ) => {
+  initializeFirebase({ useBootstrapConfig: true });
+
```

Comment:
Seems the line below is the only `getDoc` call where we read a public collection.

Could we create a special `getPublicDoc` function that always initializes and uses the bootstrap config to retrieve the tenant config?

Reply:
This is a great suggestion. Done.